users/user-creation.md

```diff
@@ -0,0 +1,42 @@
+## User Creation
+
+Aside from [automatic account creation](/jetpack/automatic-account-creation-connection.md) there is an endpoint available for creating a WordPress.com user. This endpoint assumes the user creation is part of a product provisioning flow.
+
+
+### Endpoint Information (/user)
+
+- __Method__: POST
+- __URL__:    `https://public-api.wordpress.com/rest/v1.3/jpphp/user`
```

Comment:
Similar feedback as below. Maybe we should just get rid of the alignment spaces.